src/tests/.Net4.6.2/Microsoft.PowerFx.Connectors.Tests/Microsoft.PowerFx.Connectors.Tests.csproj

```diff
@@ -0,0 +1,51 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net462</TargetFramework>
+    <OutputType>Library</OutputType>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <IsPackable>false</IsPackable>
+    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <Configurations>Debug;Release;Debug462;Debug70;DebugAll;Release462;Release70;ReleaseAll</Configurations>    
+    <DefineConstants>$(DefineConstants);$(CustomConstants)</DefineConstants>    
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- .Net 4.6.2 specifics -->
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
+    <Reference Include="System.Web" />
+    
+    <PackageReference Include="Azure.Identity" Version="1.11.0" /> <!-- CVE-2024-29992 -->
+    <PackageReference Include="Microsoft.Identity.Client" Version="4.61.1" /> <!-- CVE-2024-27086 -->
```

Comment:
Why do we need any Identity references at all?

Reply:
Not changed by this PR
This one is a security need for Azure.Identify, itself a security need for Microsoft.Data.SqlClient